common/constants/default_windows.go

```diff
@@ -0,0 +1,180 @@
+//go:build windows
```

Comment:
In this file, there are many constants that are not used by the edge, and many constants do not distinguish between Linux and Windows. Can they be divided into three files: default.go, default_linux.go, and default_windows.go, the default.go file stores public constants?

Reply:
ok,i will try

Reply:
done